### PR TITLE
show blind submissions if exists otherwise shows original notes

### PR DESCRIPTION
**Pull request by melisabok**
@pmandler this PR shows the original notes in the Program Chairs console before the submission deadline.

### CODE REVIEW

**Comment by melisabok** [comment]
it is what we usually do to allow them to see the blinded notes.

**Comment by melisabok** [comment]
it is not admin permissions in all openreview.

**Comment by pmandler** [comment]
How is it not admin permissions in all openreview? What am I missing?

**Comment by melisabok** [comment]
basically I'm adding the pc group as a member of the conference group. 
for example:

```
id: 'ICLR.cc/2019/Conference',
members: 'ICLR.cc/2019/Conference/Program_Chairs'
```

the conference group is like the 'admin' group. We create invitations and groups based on this id.

**Comment by pmandler** [comment]
If it was adding it to get_conference_id() I'd understand, but adding it to ~Super_User1 doesn't make sense to me.

**Comment by melisabok** [comment]
I'm not adding anything to the super user group, where do you see that?

**Comment by pmandler** [comment]
Line 343 of builder.py

File: openreview/conference/builder.py [hunk 1]
```diff
@@ -337,6 +339,8 @@ def open_meta_reviews(self, name, due_date = None, public = False):
 
     def set_program_chairs(self, emails):
         self.__create_group(self.get_program_chairs_id(), self.id, emails)
+        ## Give program chairs admin permissions
+        self.__create_group(self.id, '~Super_User1', [self.get_program_chairs_id()])
```
Comment (melisabok) [inline openreview/conference/builder.py]: This is saying that the signature of the Program Chairs group is the super user. Check the function parameters: 

```
def __create_group(group_id, group_owner_id, members)
```

**Comment by pmandler** [comment]
I knew I was missing something.  Thanks.

**Pull request merged**